include/seqan3/io/alignment_file/output_format_concept.hpp

```diff
@@ -95,25 +95,9 @@ SEQAN3_CONCEPT alignment_file_output_format =
 {
     t::file_extensions;
 
-    { v.write_alignment_record(stream,
-                               options,
-                               header,
-                               seq,
-                               qual,
-                               id,
-                               offset,
-                               ref_seq,
-                               ref_id,
-                               ref_offset,
-                               align,
-                               cigar,
-                               flag,
-                               mapq,
-                               mate,
-                               tag_dict,
-                               e_value,
-                               bit_score
-                               ) } -> void;
+    requires std::same_as<decltype(v.write_alignment_record(stream, options, header, seq, qual, id, offset, ref_seq,
+                                                            ref_id, ref_offset, align, cigar, flag, mapq, mate,
+                                                            tag_dict, e_value, bit_score)), void>;
```

Comment:
I prefer this "multiple parameters on one line " here too but when I wrote it, Rene said this is not our style guide.

Reply:
Our style guide also says no more than 5 parameters 😏

Reply:
:vomiting_face: